### ISSUE TITLE
can not found dowload file in download path?

### ISSUE THREAD

**Issue by chenzhenquan** [itinance/react-native-fs#488]
my download path is  :                                                                                                                            
            const downloadPath = `${RNFS.DocumentDirectoryPath}/${((Math.random() * 1000) | 0)}.apk`;
but it can not be save ,i can not found the apk file ,why?

**Comment by sandeepsinghs** [comment]
I am also facing the same issues. Some one please help.

**Comment by vitozyf** [comment]
该路径要root权限才能查看

**Comment by sandeepsinghs** [comment]
Yes. You need to ask runtime permission to access the local file system.
